### PR TITLE
contrib/beets: new package (2.0.0) + depends

### PR DESCRIPTION
**Pull request by Bridouz**
Hi cports enthusiasts,

I've juste packaged beets (CLI media manager) and all deps.

Checks are disabled for some packages where it lacks dependencies.

Hope you enjoy it.

Built and tested on arm64 (rpi400).

### CODE REVIEW

File: contrib/beets/template.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python",
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+makedepends = []
```
Comment (nekopsykose) [inline contrib/beets/template.py:12]:
```suggestion
```

delete empty lists

File: contrib/beets/template.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+pkgname = "beets"
+pkgver = "2.0.0"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python",
```
Comment (nekopsykose) [inline contrib/beets/template.py:6]:
```suggestion
```

don't need python in hostmake when other python stuff is there

File: contrib/python-confuse/template.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python",
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+makedepends = ["python-flit_core"]
```
Comment (nekopsykose) [inline contrib/python-confuse/template.py:12]:
```suggestion
    "python-flit_core",
    "python-installer",
]
```

File: contrib/python-coverage/template.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    "python-pytest-xdist",
+]
+pkgdesc = "Code coverage tool for Python"
+maintainer = "Justin Berthault <justin.berthault@zaclys.net>"
+license = "Apache-2.0"
+url = "https://coverage.readthedocs.io"
+source = f"$(PYPI_SITE)/c/coverage/coverage-{pkgver}.tar.gz"
+sha256 = "04aefca5190d1dc7a53a4c1a5a7f8568811306d7a8ee231c42fb69215571944f"
+# Some failures
+options = ["!check"]
```
Comment (nekopsykose) [inline contrib/python-coverage/template.py:27]: then skip the things that fail

File: contrib/python-jellyfish/template.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "cargo",
+    "python",
+    "python-build",
+    "python-setuptools",
+    "python-wheel",
+]
+makedepends = ["python-maturin"]
```
Comment (nekopsykose) [inline contrib/python-jellyfish/template.py:12]: this goes in hostmake and probably means you don't need setuptools

File: contrib/python-jellyfish/template.py [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+checkdepends = ["python-installer", "python-pytest"]
```
Comment (nekopsykose) [inline contrib/python-jellyfish/template.py:13]: installer goes in hostmake

File: contrib/python-jellyfish/template.py [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+source = f"$(PYPI_SITE)/j/jellyfish/jellyfish-{pkgver}.tar.gz"
+sha256 = "72aabb3bedd513cdd20712242fd51173b59972c0b146b7a0b9c6f32f1656293f"
+
+
+def do_prepare(self):
+    from cbuild.util import cargo
+
+    self.cargo = cargo.Cargo(self)
+    self.cargo.vendor()
+    cargo.setup_vendor(self)
```
Comment (nekopsykose) [inline contrib/python-jellyfish/template.py:27]: you also want

```
def init_build(self):
    from cbuild.util import cargo

    renv = cargo.get_environment(self)
    self.make_env.update(renv)
```

File: contrib/python-mediafile/template.py [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+pkgver = "0.12.0"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python",
+    "python-build",
+    "python-setuptools",
+    "python-wheel",
+]
+makedepends = ["python-flit_core"]
```
Comment (nekopsykose) [inline contrib/python-mediafile/template.py:11]: same as above, this is in the wrong place and you then don't need setuptools/wheel

File: contrib/python-mediafile/template.py [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+depends = ["mutagen", "python-six"]
+checkdepends = ["python-installer", "python-pytest"] + depends
```
Comment (nekopsykose) [inline contrib/python-mediafile/template.py:13]: same as above

File: contrib/python-mediafile/template.py [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+pkgdesc = "Handles low-level interfacing for files' tags in Python"
```
Comment (nekopsykose) [inline contrib/python-mediafile/template.py:14]:
```suggestion
pkgdesc = "Handles low-level interfacing for file tags in Python"
```

File: contrib/python-munkres/template.py [hunk 11]
```diff
@@ -0,0 +1,22 @@
+pkgname = "python-munkres"
+pkgver = "1.1.4"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python",
```
Comment (nekopsykose) [inline contrib/python-munkres/template.py:6]:
```suggestion
```

**Comment by Bridouz** [comment]
All good now beside coverage check which has some failures.

**Pull request merged**